content/people/damiri.md

```diff
@@ -0,0 +1,27 @@
+name: Delaram Amiri 
+role: student-phd
+status: Ph.D. Candidate 
+year joined: 2015
+portrait: damiri.jpg 
+badge: damiri-badge.jpg 
+interests: 
+    - Healthcare IoT 
+    -Sensor Control 
+    -Optimization 
+    -Edge Computing 
+education: 
+    - degree: M.S. in Electrical Engineering, Communications 
+        institution: Indiana University - Purdue University Indianapolis 
+        year: 2015 
+    - degree: B.S. in Electrical Engineering, Communications  
+        institution: Shiraz University 
+        year: 2013 
+email: damiri@uci.edu 
+websites:
+    -title: LinkedIn
+     url: www.linkedin.com/in/delaramamiri 
+bibliography: 
+    -title: Journal Articles items: - delaram1 -delaram2 -delaram3 -delaram4
+    -title: Conference Proceedings items: - delaram6 -delaram7
+    -title: M.Sc. Dissertation  items: -delaram5
```

Comment:
Let’s be consistent with the abbreviations of degrees, hereon sticking
to ‘M.S.’ and ‘B.S.’ instead of ‘M.Sc.’ and ‘B.Sc’, respectively.